config/src/config/safety_rules_config.rs

```diff
@@ -1,92 +1,34 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::config::SecureBackend;
 use serde::{Deserialize, Serialize};
 use std::{net::SocketAddr, path::PathBuf};
 
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 #[serde(default, deny_unknown_fields)]
 pub struct SafetyRulesConfig {
-    pub backend: SafetyRulesBackend,
```

Comment:
We pass to safety rules a node config, so let's just move the backend to be across NodeConfig since a binary shouldn't be accessing the secure backend with multiple identities.

Reply:
See my comment above.